Marv-Lous/Marv-Lous/Controllers/LoginPageViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  LoginPageViewController.swift
+//  Marv-Lous
+//
+//  Created by Erik Egers on 2022/04/01.
+//
+
+import Foundation
+import UIKit
+
+class LoginPageViewController: UIViewController {
+    
+    private lazy var loginPageViewModel = LoginPageViewModel(delegate: self)
```

Comment:
Just a small note here but normally we like to keep `@IBOutlet's` at the top of the class and then our variables 😄

Reply:
Noted, I moved it.